pkg/agent/client/client.go

```diff
@@ -311,7 +311,18 @@ func (c *client) fetchEntries(ctx context.Context) ([]*types.Entry, error) {
 	}
 	defer connection.Release()
 
-	resp, err := entryClient.GetAuthorizedEntries(ctx, &entryv1.GetAuthorizedEntriesRequest{})
+	resp, err := entryClient.GetAuthorizedEntries(ctx, &entryv1.GetAuthorizedEntriesRequest{
+		OutputMask: &types.EntryMask{
+			SpiffeId:       true,
+			Selectors:      true,
+			FederatesWith:  true,
+			Admin:          true,
+			Downstream:     true,
+			ExpiresAt:      true,
```

Comment:
I don't see `EntryExpiry` being used anywhere in the agent code. Why do we need to request it?

The utility method that converts the API proto entry to the internal proto is ignoring expiry today: https://github.com/spiffe/spire/blob/main/pkg/agent/client/util.go#L74-L83